apps/admin-gui/src/app/shared/components/hosts-list/hosts-list.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="card mt-3" [class.hide-table]="exporting" [hidden]="dataSource.filteredData.length === 0 || hosts.length === 0">
+  <div class="card-body">
+    <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true" [exporter]="exporter" class="ml-auto"></perun-web-apps-table-options>
+    <table mat-table matTableExporter [dataSource]="dataSource" #exporter="matTableExporter" matSort matSortActive="id" matSortDirection="asc" matSortDisableClear
+           class="w-100">
+
+      <ng-container matColumnDef="select">
+        <th mat-header-cell *matHeaderCellDef>
+          <mat-checkbox color="primary"
+                        (change)="$event ? masterToggle() : null"
+                        [checked]="selection.hasValue() && isAllSelected()"
+                        [indeterminate]="selection.hasValue() && !isAllSelected()"
+                        [aria-label]="checkboxLabel()">
+          </mat-checkbox>
+        </th>
+        <td mat-cell *matCellDef="let host" class="static-column-size">
+          <mat-checkbox color="primary"
+                        (click)="$event.stopPropagation()"
+                        (change)="$event ? selection.toggle(host) : null"
+                        [checked]="selection.isSelected(host)"
+                        [aria-label]="checkboxLabel(host)">
+          </mat-checkbox>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="id">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.HOSTS_LIST.TABLE_HOST_ID' | translate}}</th>
```

Comment:
"SHARED_LIB.PERUN.COMPONENTS" you have moved the component into new place. Please rewrite the translates according to it.